cabot/cabotapp/tests/test_status_check.py

```diff
@@ -171,6 +189,7 @@ def test_tcp_failure(self):
         self.assertEqual(len(checkresults), 1)
         self.assertFalse(self.tcp_check.last_result().succeeded)
         self.assertFalse(self.tcp_check.last_result().error, 'timed out')
+        self.assertEqual(list(self.tcp_check.last_result().tags.values_list('value', flat=True)), [])  # no tag...
```

Comment:
wouldn't we want `timed out` for the tag here also?

Reply:
Agreed...I didn't do this originally because I didn't like the format of the error messages for `socket.error`. I ended up stringifying the socket's errno instead.